.github/workflows/doc.yml

```diff
@@ -16,40 +31,48 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
       - name: Checkout Python
         uses: actions/setup-python@v4
         with:
           python-version: '3.10'
 
       - name: Install Python requirements
-        run: pip install -r ./docs/.utils/requirements.txt
+        run: pip install -r requirements.txt
 
       - name: Install Transifex client
         run: |
           curl -OL https://github.com/transifex/cli/releases/download/v1.6.10/tx-linux-amd64.tar.gz
           tar -xvzf tx-linux-amd64.tar.gz
 
       - name: Configure Transifex
-        run: |
-          python ./docs/.utils/transifex_utils.py
+        run: scripts/transifex_utils.py
         env:
           TX_TOKEN: ${{ secrets.TX_TOKEN }}
 
       - name: Push source files to Transifex
-        run: |
-          ./tx push
+        if: ${{ github.event_name == 'push' }}
+        run: ./tx push
         env:
           TX_TOKEN: ${{ secrets.TX_TOKEN }}
 
       - name: Pull translations from Transifex
+        if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == 'opengisch/QgisModelBaker' }}
         run: |
           ./tx pull --translations --all --minimum-perc 10
           ./tx status
         env:
           TX_TOKEN: ${{ secrets.TX_TOKEN }}
 
-      - name: Build docs
+      - name: Build documentation
```

Comment:
Do I understand correctly, that this mechanism is to build a doc-package on pull request? Cool.

Reply:
yes, you'll have a zip file so you can see results before merging